flopy/mf6/modflow/mfsimulation.py

```diff
@@ -1593,13 +1600,23 @@ def run_simulation(
             cargs : str or list of strings
                 Additional command line arguments to pass to the executable.
                 default is None
+            debug_mode : bool
+                Enable debug mode for plugins
+            kwargs : dict
+                Information passed to individual FloPy plugins.  The
+                dictionary key is the name of the FloPy plugin that should
+                receive the information.
 
         Returns
         --------
             success : bool
             buff : list of lines of stdout
 
         """
+        if debug_mode:
+            fd_debug = open("debug_run_sim.txt", "w")
```

Comment:
I think we need to think about if we want flopy to open a separate file for writing information. Users can always specify `print_input` and `print_budget` to have information written to the mf6 lst file.